CSharpMath.Forms/LatexHelper.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Xamarin.Forms;
+
+namespace CSharpMath.Forms {
+  public static class LatexHelper {
+    public static readonly string vphantom = SetColor("|", Color.Transparent);
```

Comment:
`\hphantom` is for horizontal spacing while `\vphantom` is for vertical spacing. So `hphantom` would be more appropriate here.

Reply:
The main purpose of the temporary "phantom" is, if I'm correct, to make sure that small symbols like the degree symbol are not streched. Using the vertical "|" symbol prevents that from happening. The comment in the code also says something about horizontal spacing, but I think that is not its main purpose. Besides, spacing between buttons can also be created in other ways that may be more appropriate - maybe the comment in the code is not correct.

Reply:
Hmmm. What do you suggest on horizontal spacing?

Reply:
I did not know about the \phantom command before working on this pull request, and I learned that the \phantom is a square (and \vphantom is a vertical line of zero width). If the \phantom is implemented and used instead of "|", the horizontal space will be increased. If a \vphantom is implemented, yes, then horizontal spacing must be managed another way. I think that managing the width of the button should be done by the developers using the buttons. Probably they will define the width of the buttons as a percentage of the screenwidth or something like that anyway. But if the intention is  "providing a helping hand in case the width is not managed by the developers", then probably the simple solution is the right one and using an invisible "|" is exactly that: it is related to the font size, which seems just perfect.
I have tried to find examples however where that minimum width is used. Is there an example in the example project that uses the minimum width that "|" causes?

Reply:
Leaving the "|" width both responsibilities may be better than using \phantom or \vphantom. Developers that define width themselves don't notice it, and developers that do not define width are helped without asking. In that case the comment can be updated (remove the line that mentions \phantom).

Reply:
See the Editor tab in CSharpMath.Forms.Example, I find that adding a tiny bit of horizontal space makes it look better

Reply:
Well using `\phantom` is really the intention here. `\color` with transparent is a hack.